index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+	<script src="https://kit.fontawesome.com/27ebd9ab14.js" crossorigin="anonymous"></script>
+</head>
+<body>
+	<header>
+		<nav>
+			<div class="Mylogo"> My logo</i></div>
+			<div class="openMenu"><i class="fa-solid fa-bars"></i></div>
+			<div class="closeMenu"><i class="fa-sharp fa-solid fa-xmark"></i></div>
+		</nav>
+	
+		<div class="head">
+		  <h1 class="title1">I'm Gardimy Glad to see you!</h1>
+		  
+		  <p class="title2">
+			I'm a software developer! I can help you build a product , 
+			feature or website Look through some of my work and experience! If you 
+			like what you see and have a project you need coded,
+			 don't hestiate to contact me.
+		  </p>
+		  
+		  <p class="connect"><img src="Images/LET'S CONNECT (1).png" alt="CONNECT"></p>
```

Comment:
- [ ] This text `LET'S CONNECT` should be wrapped with one of the header tags, `hx` i.e either `h3` or `h4`. This is because it serves as a header for the social media icons below it. I suggest you include it inside the `div` that wraps the social media `ul` tag.

```html
<div class="social-icon">
    <h3 class="connect"><img src="Images/LET'S CONNECT (1).png" alt="CONNECT"></h3>
     <ul>
	   <li><a href="https://twitter.com/"><img src="Images/twitter.png" alt="Twitter Icon"></a></li>
	   <li><a href="https://www.linkedin.com/in/gardimy-charles-40b795212/"><img src="Images/linkedin.png" alt="Linkedin Icon"></a></li>
	   <li><a href="#"><img src="Images/medium.png" alt="Medium Icon"></a></li>
	   <li><a href="https://github.com/Gardimy"><img src="Images/github.png" alt="GitHub Icon"></a></li>
	   <li><a href="#"><img src="Images/Fingers icon.png" alt="Fingers Icon"></a></li>
	</ul>
</div>
```